Fortinet/fortimail/_meta/fields.yml

```
@@ -0,0 +1,55 @@
+tmp.device_id:
+  name: tmp.device_id
+  description: "The device identifier"
+  type: keyword
+
+tmp.log_id:
+  name: tmp.log_id
+  description: "The log identifier"
+  type: keyword
+
+tmp.session_id:
+  name: tmp.session_id
+  description: "The session identifier"
+  type: keyword
+
+tmp.subject:
+  name: tmp.subject
+  description: "The subject"
+  type: keyword
+
+file.hash:
+  name: file.hash
+  description: "File hash"
+  type: keyword
+
+tmp.event_status:
+  name: tmp.event_status
+  description: "The event status"
+  type: keyword
+
+tmp.user_identifier:
+  name: tmp.user_identifier
+  description: "The user identifier"
+  type: keyword
+
+tmp.dns_resolution_attempt:
+  name: tmp.dns_resolution_attempt
+  description: "DNS resolution attempt"
+  type: keyword
+
```

```suggestion
file.hash:
  name: file.hash
  description: "File hash"
  type: keyword

```

It looks like your `tmp.` fields are not used in the `parser.yml` file